src/parallel/NeighbourCommunicationScheme.cpp

```diff
@@ -185,49 +185,60 @@ void DirectNeighbourCommunicationScheme::initExchangeMoleculesMPI(ParticleContai
 
 	// halo position check needs to be done, unless it is a invalidParticle returner and it had no invalid particles.
 	auto invalidParticles = moleculeContainer->getInvalidParticles();
-	
-	double rmin[DIMgeom]; // lower corner
-	double rmax[DIMgeom]; // higher corner
 
```

Comment:
This file is wrongly named! Remove "u" in "neighbor" for AE/BE consistency.

Reply:
will not do that now, there is a lot of inconsistency already...